pkg/kv/kvserver/concurrency/lock_table_test.go

```diff
@@ -1802,7 +1802,7 @@ func createRequests(index int, numOutstanding int, numKeys int, numReadKeys int)
 			lockSpans.Add(lock.None, span)
 		} else {
 			latchSpans.AddMVCC(spanset.SpanReadWrite, span, ts)
-			lockSpans.Add(lock.Intent, span)
+			lockSpans.Add(lock.Exclusive, span)
 			wi.locksToAcquire = append(wi.locksToAcquire, key)
```

Comment:
Think it's worth switching `locksToAcquire` to store `lockToAcquire` elements like we have in `workItem`, instead of `roachpb.Key` elements like we have in `benchWorkItem`? That would have make this mistake much harder to make.

Reply:
Yeah; cancelled the merge, I'll update this PR.

Reply:
Done. @nvanbenschoten could you give this a stamp as well before I re-engage bors please?